fontawesome/fontawesome.css

```diff
@@ -3537,29 +3639,3 @@ readers do not read off random characters that represent icons */
   overflow: visible;
   position: static;
   width: auto; }
-@font-face {
```

Comment:
Hi, Thanks for doing this. But removing this and the following lines breaks fontawesome.

Reply:
ok, I'll add them back and test accordingly...